Automattic/jetpack
# Lazy Images: Use srcset instead of src

#### Changes proposed in this Pull Request:

In #8879, a user suggested that we consider using a different approach for lazy images where we would rely on using `srcset` instead of `src`. This has the benefit of gracefully degrading better and I anticipate that it'll improve compatibility with plugins that were looking for the image's source.

#### Testing instructions:

- Ensure that lazy images module is on
- Create a post/page with images in it
- View source on page load and ensure that the placeholder is loaded via the `srcset` attribute
- After scrolling down, ensure the image loads properly and the `srcset` attribute now contains the actual images OR the `srcset` attribute has been remove in favor of just using `src`

#### Proposed changelog entry for your changes:
Update lazy images so that the placeholder is loaded via the srcset attribute instead of the src attribute.
